### ISSUE TITLE
Feature Suggestion

### ISSUE THREAD

**Issue by coatsy35** [Stephan-S/FS22_AutoDrive#966]
Have the ability to pause unloading with combines on the first headland run when using CP for steering. CP has this feature but it is overridden when using AD for combine unloading. This is an issue when using large combine unloading augers near trees on the first healdn run, if this feature is uninterrupted using AD then the combine would pull into pocket for unloading and not hit trees, just like it does using CP

**Comment by Iwan1803** [comment]
My suggestion: if there are trees or fences then let the combine run in the other direction. Then the pipe will be inside the field.

If i'm not wrong CP will create a parking pocket for the combine if the pipe would be outside the field. So AD will only be called if the combine is in the pocket and calls for an unloader.
This doesn't work if active unloading is activ in the AD settings of the combine.
Don't know if AD can check if the combine is in the first headland to "pause" the active unloading then.
@Axel32019 can you take a look at it?

**Comment by Tensuko** [comment]
There is an option to call the unloader on the 1st headland or not.
Off means that it will only call one when in a pocket on the 1st headland.

**Comment by coatsy35** [comment]
> There is an option to call the unloader on the 1st headland or not. Off means that it will only call one when in a pocket on the 1st headland.

Yes, this works really well . Unless you use AD and then it doesn't work. Hopefully they can link it or stop it from overriding CP's settings.

**Comment by coatsy35** [comment]
> My suggestion: if there are trees or fences then let the combine run in the other direction. Then the pipe will be inside the field.
> 
> If i'm not wrong CP will create a parking pocket for the combine if the pipe would be outside the field. So AD will only be called if the combine is in the pocket and calls for an unloader. This doesn't work if active unloading is activ in the AD settings of the combine. Don't know if AD can check if the combine is in the first headland to "pause" the active unloading then. @Axel32019 can you take a look at it?

Running in the other direction has issues as the combine is then always facing the crop with the pipe. So we don't want to do that.  I wouldn't be using active unload in this situation (in my opinion only useful for forage harvesters, not combines). So if it could pickup on CP's headland setting this would be great. This issue comes due to precall making the combine fold out the pipe as soon as the precall level is reached. It would be better if it didn't override CP and keep the pipe in until the trailer is alongside the combine (like how it works with CP). 

AD doesn't' seam to interact with CP's pockets much, or other features. Is this planned? as they have an API for it. i.e AD's trailers do not start unloading on the move when they reach a combine parked and weird stuff happens with reversing away etc... by the trailers

**Comment by Iwan1803** [comment]
Maybe you should take a look again at the AD settings: on the combine that active unloading is not active and at the unloader for the field/path settings...

**Comment by rohne83** [comment]
This issue is exact duplicate to #572
Why did you create a new issue on exact same topic even though you even commented there just one month ago?
